t/console-changejournaltype.t

```diff
@@ -66,11 +73,16 @@ foreach my $to (qw(person community)) {
     );
     $comm = LJ::load_user( $comm->user );
     is( $comm->journaltype, $types->{$to}, "Converted to a $to" );
-}
 
-### check that 'shared' is not a valid journaltype
-is(
-    $run->("change_journal_type $commname shared $owner"),
-    "error: Type argument must be 'person' or 'community'."
-);
+    if ( $comm->is_community ) {
 
+        # have to check the database directly because convenience methods
+        # know that communities aren't supposed to have passwords
+
+        my $dbh  = LJ::get_db_writer() or die "Couldn't get db master";
+        my $pass = $dbh->selectrow_array( q{SELECT password FROM password WHERE userid = ?},
```

Comment:
Shouldn't this be `password2` now?

Reply:
Here's what I couldn't figure out: you can't read the password2 table to see if the password is blank, but you also can't use the check_password method here because it immediately bails if you're trying to check the password of a community. Hence the test message saying OK if "community password is blank or not stored in password table"

Reply:
Ahh alright. ;;